DESCRIPTION

```diff
@@ -17,7 +17,7 @@ BugReports:
     https://github.com/open-systems-pharmacology/ospsuite-r/issues
 Depends: 
     R (>= 3.5),
-    rClr (>= 0.8.3)
+    rClr (>= 0.9.1)

```

Comment:
@PavelBal @Yuri05  Is this what we need to do?
Target 0.9.1 but say that we depend on R 3.5?
Appveyor will build with v4.0.x

Reply:
let's undo this